apps/bfd-server/bfd-server-launcher/pom.xml

```diff
@@ -16,7 +16,8 @@
 	</description>
 
 	<properties>
-		<jetty.version>9.4.43.v20210629</jetty.version>
+		<!-- Using Jetty 10 LTS over Jetty 11 because Jetty 11 removes support for javax.xml.bind which HAPI relies on-->
```

Comment:
slight clarification: Jetty 11 moves to the jakarta namespace for the servlet API which causes a problem for HAPI